### ISSUE TITLE
Maybe you made a typo for the key value

### ISSUE THREAD

**Issue by jezsung** [eeoom/advanced_in_app_review#1]
While looking at the implementation of this library, I found those two key values are the same. This must lead to a bug right?

``` dart
 static const String _prefKeyLaunchTimes =
      "advanced_in_app_review_launch_times";
  static const String _prefKeyRemindInterval =
      "advanced_in_app_review_launch_times";

```

**Comment by eeoom** [comment]
You are right, update on the way.

**Comment by eeoom** [comment]
Corrected, please use new version (1.0.5)
